balancer-labs/frontend-v2
# Fix pool symbol and pool name disappearing

# Description

Was seeing the 'Missing pool name or prices' tooltip pop-up where it shouldn't have and after some deeper investigation, found the root cause for the problem. This hotfix addresses that by making sure that the pool name and symbol (autogenerated) are saved BEFORE loading the final step as well as make sure that the pool name and symbol are saved on click of the 'save' button rather than relying just on the update of the model value which seemed to be hit and miss.

## Type of change


## How should this be tested?

Create a pool as normal, make sure to get to the final step. 
1. On the final step, refresh the page. It should keep the pool name and symbol as to what it was previously (autogenerated one)
2. Adjust the pool name and symbol yourself, then refresh the page. It should keep those.
3. On the final step, empty out the value for either pool name or symbol or both and make sure that an alert shows up and also disables the create pool button.

## Checklist:


### Walkthrough

(Note, you will notice in the video that on refresh of the page, the symbols for the tokens dissapear! This was an injection bug which has since been fixed)
https://www.loom.com/share/b64e4de4606c4238895910897623d2fe
